tests/vehicle_drag_test.cpp

```diff
@@ -44,17 +48,25 @@ static void clear_game_drag( const ter_id &terrain )
     here.shift( point_north );
 }
 
+/*The test setup is a vehicle with a cargo area and a fuel tank.
+The vehicle is placed on the map, facing north, at the center of the map.
+The vehicle is then filled with fuel and all cargo is removed.
+The vehicle is then turned on, and all cargo is removed.
+The vehicle is then closed, and all cargo is removed.*/
 static vehicle *setup_drag_test( const vproto_id &veh_id )
 {
+    // Clearing all vehicles from the map.    
     clear_vehicles();
+    // Creates a point on the map that is 60 x 60 units.
```

Comment:
This comment doesn't add anything, and is not correct. It's a point at 60, 60 on z-level 0, not a point that is 60x60 units.
```suggestion
```